core-services/image-mirroring/openshift/mapping_origin_4_10

```diff
@@ -43,6 +43,8 @@ registry.ci.openshift.org/origin/4.10:cluster-autoscaler-operator quay.io/opensh
 registry.ci.openshift.org/origin/4.10:cluster-baremetal-operator quay.io/openshift/origin-cluster-baremetal-operator:4.10 quay.io/openshift/origin-cluster-baremetal-operator:4.10.0 quay.io/openshift/origin-cluster-baremetal-operator:latest
 registry.ci.openshift.org/origin/4.10:cluster-bootstrap quay.io/openshift/origin-cluster-bootstrap:4.10 quay.io/openshift/origin-cluster-bootstrap:4.10.0 quay.io/openshift/origin-cluster-bootstrap:latest
 registry.ci.openshift.org/origin/4.10:cluster-capacity quay.io/openshift/origin-cluster-capacity:4.10 quay.io/openshift/origin-cluster-capacity:4.10.0 quay.io/openshift/origin-cluster-capacity:latest
+registry.ci.openshift.org/origin/4.10:cluster-capi-operator quay.io/openshift/origin-cluster-capi-operator:4.10 quay.io/openshift/origin-cluster-capi-operator:4.10.0 quay.io/openshift/origin-cluster-capi-operator:latest
+registry.ci.openshift.org/origin/4.10:cluster-capi-operator-src quay.io/openshift/origin-cluster-capi-operator-src:4.10 quay.io/openshift/origin-cluster-capi-operator-src:4.10.0 quay.io/openshift/origin-cluster-capi-operator-src:latest
```

Comment:
i missed this during review, why are we publishing a source image like this?  what does it enable?

Reply:
There is an enhancement that describes what it does https://github.com/openshift/enhancements/pull/913/files#diff-a0672895c853aefb4deaafb651c091b4e24d5dc2648e340e8e85deb4de58e564R100